tesseract-core/src/lib.rs

```diff
@@ -118,6 +118,80 @@ impl Schema {
         if query.drilldowns.is_empty() && query.cuts.is_empty(){
             return Err(format_err!("Either a drilldown or cut is required"));
         }
+
+        // The logic layer only requires the level name to be provided for a query.
+        // Here, we find the dimension and hierarchy names for the given level names.
+        // NOTE: Failing silently for queries with multiple drilldowns if not all of
+        //       the level names are found.
+        let cube_obj = self.cubes.iter()
+            .find(|c| &c.name == &cube)
+            .ok_or(format_err!("Could not find cube"))?;
+
+        let mut drilldowns: Vec<Drilldown> = vec![];
+        let mut cuts: Vec<Cut> = vec![];
+        let mut properties: Vec<Property> = vec![];
```

Comment:
Should we omit properties? At least I'm not sure how to specify properties, since they also need an associated level drilldown.

Reply:
I talked to Dave and he told me the current logic layer requires only the property name and figures the rest out. So that's what I implemented here.